BuildResidentialScheduleFile/measure.rb

```diff
@@ -147,7 +147,7 @@ def run(model, runner, user_arguments)
 
       # output csv path
       args[:output_csv_path] = "#{filename}.csv"
-      args[:output_csv_path] = "#{filename}_#{i + 1}.csv" if i > 0
+      args[:output_csv_path] = "#{filename}_#{i + 1}.csv" if i > 0 && args[:building_id].get == 'ALL'
```

Comment:
I found this while changing how the sample files are created; they now call this measure multiple times with a single building ID. And when a single building ID is provided, the filename provided should be used directly.